src/shared/services/fields/fields.purger.ts

```diff
@@ -37,23 +37,23 @@ export class FieldsPurger {
       let case_field = this.findCaseFieldByWizardPageFieldId(currentPage, wpf);
       if (this.hasShowConditionField(case_field, formFields)) {
         let condition = new ShowCondition(case_field.show_condition);
-        if (this.isHidden(condition, formFields) && !(this.isReadonly(case_field))) {
+        if (this.isHidden(condition, formFields.data) && !(this.isReadonly(case_field))) {
           this.resetField(form, case_field);
         }
       }
     });
   }
 
   private isHidden(condition, formFields) {
-    return !condition.match(formFields.data);
+    return !condition.match(formFields);
   }
 
   private findCaseFieldByWizardPageFieldId(currentPage, wizardPageField) {
     return currentPage.case_fields.find(cf => cf.id === wizardPageField.case_field_id);
   }
 
   private hasShowConditionPage(wizardPage, formFields): boolean {
-    return wizardPage.show_condition && formFields.data[this.getShowConditionKey(wizardPage.show_condition)];
+    return wizardPage.show_condition && formFields[this.getShowConditionKey(wizardPage.show_condition)];
```

Comment:
what's the reason of this change?

Reply:
We are directly getting data fields. We don't have to call again data property.

Reply:
ok got it cheers :)